DJHelper/DJHelper/Controller/Song Controller.swift

```diff
@@ -257,18 +257,22 @@ class SongController {
         }
     }
     // MARK: - Delete Song from Playlist
-    func deleteSongFromPlaylist(track: TrackResponse, completion: @escaping (Result<(), SongError>) -> Void) {
+    func deleteSongFromPlaylist(song: Song, completion: @escaping (Result<(), SongError>) -> Void) {
          guard let bearer = Bearer.shared.token else {
              print("Error on line: \(#line) in function: \(#function)\n")
              //CHANGE ERROR
              completion(.failure(.noEventsInServerOrCoreData))
              return
              }
-
+        guard let trackResponse = song.songToTrackResponse else {
```

Comment:
Good work figuring out the data needed for the endpoint and converting the request to match.